app/web_hub.go

```diff
@@ -359,7 +359,7 @@ func (h *Hub) Register(webConn *WebConn) {
 func (h *Hub) Unregister(webConn *WebConn) {
 	select {
 	case h.unregister <- webConn:
-	case <-h.didStop:
+	case <-h.stop:
```

Comment:
Later, I'll investigate whether we really ought to flip this over to `h.stop` uniformly in this file, vs. the weirdness that is `h.didStop` for some interactions and `h.stop` for others, but for now this restores the original behaviour.